packages/grafana-ui/src/components/TimePicker/TimePickerPopover.tsx

```diff
@@ -5,7 +5,7 @@ import { Moment } from 'moment';
 import { TimePickerCalendar } from './TimePickerCalendar';
 import { TimePickerInput } from './TimePickerInput';
 import { mapTimeOptionToTimeRange } from './time';
-import { Timezone } from '../../../../../public/app/core/utils/datemath';
+import { Timezone } from '@grafana/ui/src/utils/datemath';
```

Comment:
don't think internal components should refer to it's own files using it's package name